pulumi/python/kubernetes/nginx/ingress-controller/__main__.py

```diff
@@ -218,14 +219,13 @@ def namespace_by_name(name):
                                                                               provider=k8s_provider))
 
 pstatus = kic_chart.status
-
+pstatus.apply(lambda status: pulumi.log.info(f"[DEBUG]: Helm chart status: {status}"))
 srv = Service.get(resource_name="nginx-ingress",
-                  id=Output.concat("nginx-ingress", "/", pstatus.name, "-nginx-ingress"),
+                  id=Output.concat("nginx-ingress", "/", pstatus.name, "-nginx-ingress-controller"),
```

Comment:
**Annotation**:
Necessary to align with the new name of the controller in new helm chart